<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_signUp">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(23, 231, 183);
    border: none;
	border-radius: 10px;
}

QPushButton:hover {
    background-color: rgba(23, 231, 183, 150);
}
</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_login">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(23, 231, 183);
    border: none;
    border-radius: 10px;
}

QPushButton:hover {
	background-color: rgba(23, 231, 183, 150);
}
</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_name_entry_login">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_entry_login">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="user_name_label_login">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>User Name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_label_login">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Password :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Inicie_mode_label_login">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Inicie Mode</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_voice_login">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Voice</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_manual_login">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Manual</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../image/lucy_logo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x3:1, y2:2, stop:0 rgba(10, 38, 31, 255), stop:1 rgba(1, 229, 161, 255));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_login">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_login">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>btn_signUp</zorder>
   <zorder>btn_login</zorder>
   <zorder>user_name_entry_login</zorder>
   <zorder>password_entry_login</zorder>
   <zorder>user_name_label_login</zorder>
   <zorder>password_label_login</zorder>
   <zorder>Inicie_mode_label_login</zorder>
   <zorder>radioButton_manual_login</zorder>
   <zorder>radioButton_voice_login</zorder>
   <zorder>label</zorder>
   <zorder>listWidget_login</zorder>
   <zorder>checkBox_login</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
